packages/synapse-react-client/src/components/PortalList/README.md

```diff
@@ -0,0 +1,138 @@
+# PortalList Component
+
+A component for displaying and managing Synapse Portals with infinite scroll pagination.
+
+## Overview
+
+The `PortalList` component provides a responsive grid layout for displaying Synapse Portal information. It uses infinite scroll to load additional portals as the user scrolls down the page.
+
+## Components
+
+### PortalList
+
+The main container component that handles fetching and displaying the list of portals.
+
+**Props:**
+
+- `title?: string` - Optional title for the portal list (default: "Synapse Portals")
+- `emptyMessage?: string` - Message to display when no portals are found (default: "No portals found.")
+
+**Example:**
+
+```tsx
+import { PortalList } from '@sage-bionetworks/synapse-react-client'
+
+function MyComponent() {
+  return (
+    <PortalList
+      title="All Available Portals"
+      emptyMessage="No portals are currently available."
+    />
+  )
+}
+```
+
+### PortalCard
+
+A card component that displays information about a single portal.
+
+**Props:**
+
+- `portal: Portal` - The portal object to display
+
+**Portal Properties:**
+
+- `id` - Unique ID of the portal
+- `name` - Portal name
+- `url` - Portal base URL
+- `createdBy` - User ID who created the portal
+- `createdOn` - Creation timestamp
+- `modifiedBy` - User ID who last modified the portal
+- `modifiedOn` - Last modification timestamp
+- `etag` - Entity tag for optimistic concurrency control
+
+**Example:**
+
+```tsx
+import { PortalCard } from '@sage-bionetworks/synapse-react-client'
+import { Portal } from '@sage-bionetworks/synapse-client'
```

Comment:
```suggestion
import { PortalCard } from 'synapse-react-client'
import { Portal } from '@sage-bionetworks/synapse-client'
```

Reply:
Much of the README was out of date, removed